spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapConfigProperties.java

```diff
@@ -72,10 +77,16 @@ public void setSources(List<Source> sources) {
 	 */
 	public List<NormalizedSource> determineSources() {
 		if (this.sources.isEmpty()) {
-			return Collections.singletonList(new NormalizedSource(name, namespace));
+			if (useNameAsPrefix) {
+				LOG.warn(
```

Comment:
just trying to catch an edge case where a miss-configuration might be hard to debug.